bases/rsptx/interactives/runestone/activecode/js/activecode-i18n.en.js

```diff
@@ -111,6 +111,7 @@ $.i18n().load({
             "Compiling and Running your Code Now...",
         msg_activecode_server_comm_err: "Error communicating with the server.",
         msg_activecode_save_run: "Save & Run",
+        msg_activecode_run_button_title: "Save & Run (ctrl-s)",
```

Comment:
Is there another way to make the ctrl-s discoverable?  This seems like it is going to make the save & run button a lot wider.